pulpcore/cli/rpm/context.py

```diff
@@ -103,6 +103,14 @@ class PulpRpmRemoteContext(PulpRemoteContext):
     }
 
 
+class PulpUlnRemoteContext(PulpRemoteContext):
+    ENTITY = _("uln remote")
+    ENTITIES = _("uln remotes")
+    HREF = "rpm_uln_remote_href"
+    ID_PREFIX = "remotes_rpm_uln"
+    NULLABLES = PulpRpmRemoteContext.NULLABLES + {"uln_server-base_url"}
```

Comment:
Pending on the outcome of the other discussion, but we should either not have this line or use that version:
```suggestion
    NULLABLES = PulpRemoteContext.NULLABLES + {"uln_server-base_url"}
```